File: LOO 2°Parcial/Simulador de Vehiculos CSharp/Simulador de Vehiculos CSharp/DemasiadoRapidoException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simulador_de_Vehiculos_CSharp
{
    class DemasiadoRapidoException : Exception
    {
        public DemasiadoRapidoException() :base ("\nEl vehiculo llego a su velociad maxima.")
        {
            //("\nEl vehiculo llego a su velociad maxima.");
        }

        //@Override
        public String toString()
        {
            return base.ToString();
        }
    }
}

File: LOO 2°Parcial/Simulador de Vehiculos Java/src/Clases/DemasiadoRapidoException.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Clases;

/**
 * 
 * @author Dimitri Medina Novelo
 */
public class DemasiadoRapidoException extends Exception{
    
    public DemasiadoRapidoException(){
      super("\nEl vehiculo llego a su velociad maxima.");
    }
    
    @Override
    public String toString(){
      return super.toString();
    }
}
